java-frontend/src/main/java/org/sonar/java/se/FlowComputation.java

```diff
@@ -176,7 +178,8 @@ private void run(@Nullable final ExplodedGraph.Node node, @Nullable final Symbol
     }
     MethodYield selectedMethodYield = currentNode.selectedMethodYield(parent);
     if (selectedMethodYield != null) {
-      selectedMethodYield.flow(argIdx).forEach(flowBuilder::add);
+      // FIXME
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1134)